content/community.rst

```diff
@@ -10,26 +13,147 @@ after:
+      Besides being active participants in the Sass community,
+      we love the thoughtfulness and dedication to diversity
+      that the SassConf organizers bring to the community.
+  - name: 'Black Girls CODE'
+    url: 'http://blackgirlscode.com/'
+    type: 'donations'
+    years: '2013–2016'
+    description: |
+      has set out to prove to the world
+      that girls of every color have the skills
+      to become the programmers of tomorrow.
+      By promoting classes and programs
+      they help to grow the number of women of color
+      working in technology,
+      and give underprivileged girls a chance
+      to become the masters of their technological worlds.
+  - name: 'Thunder Valley Community Development Corporation'
+    url: 'http://thundervalley.org/'
+    type: 'donations'
+    years: '2014, 2015'
+    description: |
+      is a Lakota-run, grassroots Community Development Corporation
+      that is building a community for systemic change
+      on the Pine Ridge Reservation in South Dakota.
+  - name: 'PyLadies'
+    url: 'http://www.pyladies.com/'
+    type: 'donations'
+    years: '2012, 2014, 2015'
+    description: |
+      is an international mentorship group
+      with a focus on helping more women become active participants
+      and leaders in the Python open-source community.
+  - name: 'Django Girls'
+    url: 'https://djangogirls.org/'
+    type: 'donations'
+    years: '2015–2017'
+    description: |
+      is a non-profit organization
+      and a community that empowers and helps women to organize free,
+      one-day programming workshops by providing tools,
+      resources and support.
+  - name: 'PyCon'
+    url: 'https://us.pycon.org/'
+    type: 'sponsorships'
+    years: '2013, 2014'
+    description: |
+      is the largest annual gathering
+      for the people who use and develop
+      the open-source Python programming language.
+  - name: 'The Mixin'
+    url: 'http://themixinsf.com/'
+    type: 'sponsorships'
+    years: '2015'
+    description: |
+      is a San Francisco Sass front-end meetup.
+  - name: 'COSOLA-MACILE'
+    type: 'donations'
+    years: '2013, 2014'
+    description: |
+      is a non-profit organization
+      focused on K-12 STEM education
+      in less advantaged communities.
+  - name: 'Ada Initiative'
+    url: 'https://adainitiative.org/'
+    type: 'donations'
+    years: '2012, 2013'
+    description: |
+      supported women in open technology and culture
+      through activities such as producing codes of conduct
+      and anti-harassment policies,
+      advocating for gender diversity,
+      and teaching ally skills.
+summary: |
+  We love being part of
+  a vibrant web design and development community.
+  We benefit daily from the generous contributions of others,
+  and believe that a `diverse network`_ of shared experiences and ideas
+  makes all of our work stronger.
+  In addition to our `open-source software`_ and `public speaking`_,
```

Comment:
Maybe a bit controversial, but is it necessary to hyphenate open source? I know [Github doesn‘t](https://github.com/open-source)